.env.test

```diff
@@ -19,6 +19,7 @@ SQS_DEPLOYMENT_QUEUE_REGION=us-west-1
 SQS_BRANCH_QUEUE_URL=http://127.0.0.1:4566/000000000000/test-branch
 SQS_BRANCH_QUEUE_REGION=us-west-1
 BULL_QUEUE_PREFIX=bulltest
+DEBUG=nock.*
```

Comment:
Added this because degubbing nock (and trying to figure out how to debug nock) is laaaaaaaaame. It does create noise in the terminal but... who cares? It means we actually can figure out why nock tests fail. Without it, we have zero insight into why a mock request fails.

Reply:
agreed, is a nightmare without this!